src/main/java/storage/Storage.java

```diff
@@ -16,8 +16,8 @@
 
 public class Storage {
 
-    private static Logger logger = Logger.getLogger("Storage");
     public static final Path FILE_PATH = Path.of("./save/tasks.txt");
+    private static Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
```

Comment:
Thanks for fixing the checkstyle error! @kyhjonathan do take note for future work!